### DISCUSSION TITLE
WARNING - Missing GitHub access token

### DISCUSSION THREAD

**Discussion by sameeshem** [microsoft/OSSGadget#356] [Q&A]
Hello,
i am getting an error while using the oss-health tool. can you please help how to setup GITHUB_ACCESS_TOKEN while using docker container

**Comment by scovetta** [comment]
Hi @sameeshem!
You can pass that environment variable using `-e GITHUB_ACCESS_TOKEN=abcdefg` when you run Docker. I've added an example to the [wiki](https://github.com/microsoft/OSSGadget/wiki/Docker-Image) page.

But also keep in mind, we haven't made many recent updates to OSS Health -- you might find the OpenSSF [Scorecard](https://github.com/ossf/scorecard) project to be much better at this time.

**Reply by sameeshem** [reply]
Thanks Michael for adding the info in WIKI. 
Sure i will keep that in mind about OSS Scorecard. i am just evaluating the OSS Health as of now.
